api/apiFunctions.enum.ts

```diff
@@ -41,35 +41,35 @@ export enum Document {
  * Enum representing NFL teams with their corresponding unique identifiers.
  */
 export enum NFLTeams {
-  '49ERS' = '49ers',
-  BEARS = 'Bears',
-  BENGALS = 'Bengals',
-  BILLS = 'Bills',
-  BRONCOS = 'Broncos',
-  BROWNS = 'Browns',
-  BUCCANEERS = 'Buccaneers',
-  CARDINALS = 'Cardinals',
-  CHARGERS = 'Chargers',
-  CHIEFS = 'Chiefs',
-  COMMANDERS = 'Commanders',
-  COWBOYS = 'Cowboys',
-  DOLPHINS = 'Dolphins',
-  EAGLES = 'Eagles',
-  FALCONS = 'Falcons',
-  GIANTS = 'Giants',
-  JAGUARS = 'Jaguars',
-  JETS = 'Jets',
-  LIONS = 'Lions',
-  PACKERS = 'Packers',
-  PANTHERS = 'Panthers',
-  PATRIOTS = 'Patriots',
-  RAIDERS = 'Raiders',
-  RAMS = 'Rams',
-  RAVENS = 'Ravens',
-  SAINTS = 'Saints',
-  SEAHAWKS = 'Seahawks',
-  STEELERS = 'Steelers',
-  TEXANS = 'Texans',
-  TITANS = 'Titans',
-  VIKINGS = 'Vikings',
+  '49ers' = '49ers',
```

Comment:
@shashilo was the reason for changing the key to a lowercase string to match the document ID?

Reply:
Correct.